src/Surfnet/StepupSelfService/SamlStepupProviderBundle/DependencyInjection/SurfnetStepupSelfServiceSamlStepupProviderExtension.php

```diff
@@ -119,9 +138,11 @@ private function loadProviderConfiguration(
             $configuration['view_config']['explanation'],
             $configuration['view_config']['authn_failed'],
             $configuration['view_config']['pop_failed'],
-        ]);
+            ]
+        );
         $viewConfigDefinition->addTag(self::VIEW_CONFIG_TAG_NAME);
-        $viewConfigDefinition->setPublic(true);
+        // Stop making the service public, use the ViewConfigContainer instead
+        $viewConfigDefinition->setPublic(false);
```

Comment:
I'm not fully certain. But not setting it to be public will automatically make it private. As this became the default in SF 4 IIRC.